### PR TITLE
Css chatroom/ Index vs show pages of Author and Category

### PR DESCRIPTION
**Pull request by hungngotai**

### CODE REVIEW

File: app/views/authors/_author.html.slim [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.col-md-4.category_item
+  .card.portfolio-box
+    a href="#{author_path(author)}"
```
Comment (N-Viet-ruby-dev) [inline app/views/authors/_author.html.slim]: dùng link_to

File: app/views/categories/_category_item.html.slim [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.col-md-4.category_item
+  .card.portfolio-box data-id="#{category_item.id}"
+    a href="#{category_path(category_item)}"
```
Comment (N-Viet-ruby-dev) [inline app/views/categories/_category_item.html.slim]: Dùng link_to

File: app/views/authors/_author.html.slim (hunk 1 above)
Comment (hungngotai) [reply app/views/authors/_author.html.slim]: updated

File: app/views/categories/_category_item.html.slim (hunk 2 above)
Comment (hungngotai) [reply app/views/categories/_category_item.html.slim]: updated

**Pull request merged**